src/cloud-api-adaptor/Makefile

```diff
@@ -7,18 +7,18 @@ include Makefile.defaults
 
 SHELL = bash -o pipefail
 
-ARCH        ?= $(subst x86_64,amd64,$(shell uname -m))
+ARCH         ?= $(subst x86_64,amd64,$(shell uname -m))
 # Default is dev build. To create release build set RELEASE_BUILD=true
 RELEASE_BUILD ?= false
 # CLOUD_PROVIDER is used for runtime -- which provider should be run against the binary/code.
 CLOUD_PROVIDER ?=
-GOOPTIONS   ?= GOOS=linux GOARCH=$(ARCH) CGO_ENABLED=0
-GOFLAGS     ?=
-BINARIES    := cloud-api-adaptor agent-protocol-forwarder process-user-data
-SOURCEDIRS  := ./cmd ./pkg
-PACKAGES    := $(shell go list $(addsuffix /...,$(SOURCEDIRS)))
-SOURCES     := $(shell find $(SOURCEDIRS) -name '*.go' -print)
-ATTESTER    ?= none
+GOOPTIONS    ?= GOOS=linux GOARCH=$(ARCH) CGO_ENABLED=0
+GOFLAGS      ?=
+BINARIES     := cloud-api-adaptor agent-protocol-forwarder process-user-data
+SOURCEDIRS   := ./cmd ./pkg
+PACKAGES     := $(shell go list $(addsuffix /...,$(SOURCEDIRS)))
+SOURCES      := $(shell find $(SOURCEDIRS) -name '*.go' -print)
+TEE_PLATFORM ?= none
```

Comment:
I don't think it needs to define `TEE_PLATFORM` here because it will be inherited from Makefile.defaults